src/datatypes/interval.js

```diff
@@ -42,6 +42,9 @@ class Interval {
     if (this.highClosed && this.high != null && cmp.equals(this.high, item)) {
       return true;
     }
+    if (this.low == null && this.high == null) {
+      return false;
+    }
```

Comment:
So... I think you did this to satisfy this test:
```
"TestInNull": Tuple{
    expression: 5 in Interval[null, null],
    output: false
  }
```
But now that I look at it, the test is just flat out wrong.  In CQL, null boundaries have a different meaning depending on if the boundary is closed or not.  A closed null boundary means "infinite", whereas an open null boundary means "unknown".  So, if we have `Interval[null, null]` (closed boundaries), then technically _everything_  should be contained by that.

The documentation for [Interval#in](http://build.fhir.org/ig/HL7/cql/09-b-cqlreference.html#in) even suggests this:
> For closed interval boundaries, if the interval boundary is null, the result of the boundary comparison is considered true.

So, I think we need to back out this change and for keep the test ignored, but add it to the section of ignored tests pending feedback from Bryn.  I'll reach out to him to make sure I'm not missing something.

Reply:
Update: Bryn did confirm that the test is wrong.  The answer should be "true" and the name of the test should actually be "TestInNullBoundaries".  So can you just change the `false` to `true` in this test (and still back out the change in the logic so that the test passes)?